fe/fe-core/src/main/java/org/apache/doris/alter/SchemaChangeJobV2.java

```diff
@@ -897,6 +903,10 @@ private void replayPendingJob(SchemaChangeJobV2 replayedJob) throws MetaNotFound
         olapTable.writeLock();
         try {
             addShadowIndexToCatalog(olapTable);
+            if (invertedIndexChange) {
```

Comment:
All schema change job will set table's state to SCHEMA_CHANGE.
Why we have a `if` here?

Reply:
othre shcema change job set table's state to SCHEMA_CHANGE in function `replayCreateJob`, invertedIndexChange job begin with WAITING_TXN, no PENDING state, so here need set table state to SCHEMA_CHANGE when replay job begin at WAITING_TXN